### DISCUSSION TITLE
Can we have a statistic for enemies' party size?

### DISCUSSION THREAD

**Discussion by MakiseKurisu** [Heroes-Profile/heroesprofile#1057] [Ideas]
I feel like I'm getting matched against 5-man way more often than what Party Stat suggests they should occur by average.

Few things I thinks are helpful:

* List `Enemy Party Size Win Rates` under `Party Size Win Rates`. Only group by the largest enemy party size per game.
* In `Party Stat`, also include `Total Games` for each row, so it is easier to calculate the chance facing a specific enemy party composition (read 5-man) given our current party composition, and the win rate (if there is any doubt).
* Potentially, add a graph showing the count of the games grouped by the largest party size on each team (so each game counts twice for both sides) over time, so we know when the 5 stacks are out of the wood.

**Comment by Zemill** [comment]
> In Party Stat, also include Total Games for each row, so it is easier to calculate the chance facing a specific enemy party composition (read 5-man) given our current party composition, and the win rate (if there is any doubt).

What page are you looking at?  I see totals on the Global/Party page, and on the Profile page
